src/rgw/rgw_file.h

```diff
@@ -762,6 +751,10 @@ namespace rgw {
 	lock_guard guard(mtx);
 	events.push_back(ev);
       }
+
+      void push_event(const event& ev, unique_lock& guard) {
```

Comment:
@mattbenjamin method is a bit confusing, as guard is not used here. Maybe just call it differently?